cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java

```diff
@@ -1520,12 +1534,31 @@ private TSStatus processPartitionedPlan(PhysicalPlan plan) throws UnsupportedPla
     } catch (MetadataException e) {
       logger.error("Cannot route plan {}", plan, e);
     }
-    // the storage group is not found locally, forward it to the leader
+    // the storage group is not found locally
     if (planGroupMap == null || planGroupMap.isEmpty()) {
-      logger.debug("{}: Cannot found storage groups for {}", name, plan);
+      if (plan instanceof InsertPlan && IoTDBDescriptor.getInstance().getConfig()
```

Comment:
How about CreateTimeSeriesPlan? and other plans which can cause first check storage group exist or not ?

Reply:
The problem you mentioned refers to `CreateTimeseriesPlan`, `InsertTimeseriesPlan`, `DeleteTimeseriesPlan` and `DeleteTimeseriesPlan`. However, if storage group is not set when executing `DeleteTimeseriesPlan` and `DeleteTimeseriesPlan`, logically there's no necessity for the storage group to be created. As in the method `executeNonQuery` in `MetaGroupMember`, these two plans will trigger `processNonPartitionDataPlan` and `processNonPartitionMetaPlan` respectively. `CreateTimeseriesPlan` and `InsertTimeseriesPlan` are "partitioned plans". As a result, if storage group is not set for these 2 types, auto creation will be performed.